K6 crashes with "panic: runtime error: integer divide by zero" when run in CodeFresh

## Environment
<!--- Provide as much information about your environment as possible: output from `k6 version`,
      OCI runtime and image information if using containers, cloud execution environment, etc. -->
- k6 version: 0.27.0 / 0.27.1
- OS and version: CodeFresh
- Docker version and image, if applicable: loadimpact/k6:0.27.0 and loadimpact/k6:0.27.1


## Expected Behavior
<!--- Tell us what should happen -->
k6 should successfully run load tests in a container

## Actual Behavior
<!--- Tell us what happens instead of the expected behavior -->
k6 crashes with the following stack trace:
```
execution: local                                                                                                                                                                                                               
     script: /scripts/debug.bundle.js                                                                                                                                                                                            
     output: json=test_results.json                                                                                                                                                                                              
                                                                                                                                                                                                                                 
  scenarios: (100.00%) 1 executors, 10 max VUs, 1m30s max duration (incl. graceful stop):                                                                                                                                        
           * default: Up to 10 looping VUs for 1m0s over 3 stages (gracefulRampDown: 30s, gracefulStop: 30s)                                                                                                                     
                                                                                                                                                                                                                                 
panic: runtime error: integer divide by zero--] Init VUs                                                                                                                                                                         
                                                                                                                                                                                                                                 
goroutine 6 [running]:                                                                                                                                                                                                           
github.com/loadimpact/k6/cmd.renderMultipleBars(0xc007600101, 0x7, 0x0, 0x0, 0xc002678000, 0x2, 0x2, 0x5327a4, 0xd1fce0, 0x0)                                                                                                    
        github.com/loadimpact/k6/cmd/ui.go:218 +0xda9                                                                                                                                                                            
github.com/loadimpact/k6/cmd.showProgress.func2(0xc003713101)                                                                                                                                                                    
        github.com/loadimpact/k6/cmd/ui.go:283 +0x8c                                                                                                                                                                             
github.com/loadimpact/k6/cmd.showProgress(0x10d8a40, 0xc000d94100, 0x0, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc00159cd80, ...)                                                                                                         
        github.com/loadimpact/k6/cmd/ui.go:352 +0x52a                                                                                                                                                                            
github.com/loadimpact/k6/cmd.glob..func11.1(0x10d8a40, 0xc000d94100, 0xc0005d6300, 0xc0009a2500, 0xc0001b2000, 0xc000ec4bd0)                                                                                                     
        github.com/loadimpact/k6/cmd/run.go:190 +0xb9                                                                                                                                                                            
created by github.com/loadimpact/k6/cmd.glob..func11                                                                                                                                                                             
        github.com/loadimpact/k6/cmd/run.go:189 [+0xd34]
```

## Steps to Reproduce the Problem
This seems similar to https://github.com/loadimpact/k6/issues/1559 as it is unable to get the terminal width correctly, but 0.27.1 did not fix the issue in CodeFresh, and still crashes in the same way.  Reverting to 0.26.2 works as expected.

Attempting to run any script on a k6 0.27.0 or 0.27.1 container in CodeFresh should see this behavior.

--- a/cmd/ui.go
+++ b/cmd/ui.go
@@ -254,11 +254,15 @@ func showProgress(
 	}
 
 	var errTermGetSize bool
-	termWidth, _, err := terminal.GetSize(int(os.Stdout.Fd()))
-	if err != nil && stdoutTTY {
-		logger.WithError(err).Warn("error getting terminal size")
-		termWidth = defaultTermWidth
-		errTermGetSize = true
+	termWidth := defaultTermWidth
+	if stdoutTTY {
+		tw, _, err := terminal.GetSize(int(os.Stdout.Fd()))
+		if !(tw > 0) || err != nil {
+			errTermGetSize = true
+			logger.WithError(err).Warn("error getting terminal size")
+		} else {
+			termWidth = tw
+		}
 	}
 
 	// Get the longest left side string length, to align progress bars
@@ -333,19 +337,19 @@ func showProgress(
 			outMutex.Unlock()
 			return
 		case <-winch:
-			if !errTermGetSize {
+			if stdoutTTY && !errTermGetSize {
 				// More responsive progress bar resizing on platforms with SIGWINCH (*nix)
-				termWidth, _, err = terminal.GetSize(fd)
-				if err != nil {
-					termWidth = defaultTermWidth
+				tw, _, err := terminal.GetSize(fd)
+				if tw > 0 && err == nil {
+					termWidth = tw
 				}
 			}
 		case <-ticker.C:
 			// Default ticker-based progress bar resizing
-			if !errTermGetSize && winch == nil {
-				termWidth, _, err = terminal.GetSize(fd)
-				if err != nil {
-					termWidth = defaultTermWidth
+			if stdoutTTY && !errTermGetSize && winch == nil {
+				tw, _, err := terminal.GetSize(fd)
+				if tw > 0 && err == nil {
+					termWidth = tw
 				}
 			}
 		}

